src/Dnn.Modules.Vendors/Dnn.Modules.Vendors.csproj

```diff
@@ -46,22 +46,15 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="DotNetNuke, Version=8.0.0.809, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>packages\DotNetNuke.Core.8.0.0.809\lib\net40\DotNetNuke.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\..\DNN9\bin\DotNetNuke.dll</HintPath>
```

Comment:
I tried to update it to DNN9 but I had to leave it as DNN8. I got those references resolved (hopefully). When I tried to go to the latest version of DNN 9.10.2 I was not able to get it to update because of a hard reference to Microsoft.Extensions.DependencyInjection 2.1.1 by DotNetNuke.DependencyInjection even though it says the dependency is >= 2.1.1 of Microsoft.Extensions.DependencyInjection